src/components/TextInput/TextInput.stories.tsx

```diff
@@ -107,6 +108,22 @@ export const WithControls: Story = {
     },
     jest: "TextInput.test.tsx",
   },
+  play: async ({ canvasElement }) => {
+    const textInput = within(canvasElement).getByRole("textbox");
+    await userEvent.click(textInput);
+    await userEvent.type(textInput, "Hello World");
+
+    await waitFor(() => {
+      expect(textInput).toHaveValue("Hello World");
+    });
```

Comment:
In the list of interactions in Storybook, this shows as `waitFor(anonymous)`.  Is it possible to show more infomration about what is happening within the `waitFor` method?